src/ui/public/typeahead/partials/typeahead-items.html

```diff
@@ -5,7 +5,10 @@
   class="typeahead-items"
   >
 
+  <div ng-if="!typeahead.query.length" ng-transclude></div>
```

Comment:
I think it's better than hiding it. It's definitely not perfect though... could use some of @snide's magic.